gpii/node_modules/transformer/src/js/Transformer.js

```diff
@@ -62,6 +111,43 @@ fluid.registerNamespace("gpii.transformer");
         return settingsHandler;
     };
 
+    /**
+     * Remove given values from an array.
+     *
+     * @param {Array} array - An array to have given values removed - MODIFIED IN PLACE.
+     * @param {String|Array} - A string, or an array of strings, to be removed.
```

Comment:
It doesn't look like they actually need to be strings - it looks like any primitive value supporting === will do

Reply:
Bump comment